src/App.js

```diff
@@ -1,27 +1,26 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
 
 class App extends Component {
   render() {
     return (
       <div className="App">
-        <img 
-          src={require('./images/mentor.jpg')} 
-          alt="Mentor" 
-          className="App__background-image"
-        />
-        <header className="App__header">
-          <h1 className="App__title">Coding Coach</h1>
-          <h4 className="App__tagline">Connecting developers with mentors worldwide.</h4>
-          <button className="button button__white--fill">
-            <a 
-              href="https://docs.google.com/forms/d/1TraoygcJay39cLiBQM-ROlToy7xIgcdx_VvCQuE1lJw/edit#responses"
-              target="blank"
-              className="button__link"
-            >
-              Get Started
-            </a>
-          </button>
-        </header>
+        <section className="App__hero">
```

Comment:
What do you think if we switch to lowercase early on?

Reply:
For components? They should be uppercase.

Reply:
You mean writing CSS classes upper case, because it's a "component"? (Just asking, never heard of this approach).

Reply:
Yeah. When you defines styles for a component, capitalize the definition so it is consistent with how its written in JSX :p